src/commands/components/dev/test.ts

```diff
@@ -138,21 +140,40 @@ const valuesFromAnswers = ({
 
 export default class TestCommand extends Command {
   static description =
-    "Run a Component in Prismatic by publishing it into a test Integration";
+    "Run an action of a component within a test integration in the integration runner";
   static flags = {
     envPath: Flags.string({
       required: false,
       default: ".env",
       char: "e",
       description: "Path to dotenv file to load for supplying testing values",
     }),
+    "no-build": Flags.boolean({
+      required: false,
+      default: false,
+      char: "n",
+      description: "Skip building the component prior to testing",
+    }),
```

Comment:
I think the `oclif` convention for this is to use the `allowNo` flag since it allows you to keep the later flag checks the same as other boolean flags:
```suggestion
    build: Flags.boolean({
      required: false,
      default: true,
      allowNo: true,
      char: "n",
      description: "Build the component prior to testing",
    }),
```

Reply:
Good call - thanks!